### PR TITLE
Update rrggbbaa spec

### PR DESCRIPTION
**Pull request by lens0021**

### CODE REVIEW

**Comment by saschanaz** [comment]
Do we really need to have this when there is the corresponding MDN item? https://caniuse.com/mdn-css_types_color_alpha_hexadecimal_notation

File: features-json/css-rrggbbaa.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
   "title":"#rrggbbaa hex color notation",
   "description":"The CSS Color Module Level 4 defines new 4 & 8 character hex notation for color to include the opacity level.",
-  "spec":"https://drafts.csswg.org/css-color/#hex-notation",
-  "status":"unoff",
+  "spec":"https://www.w3.org/TR/css-color-4/#hex-color",
```
Comment (saschanaz) [inline features-json/css-rrggbbaa.json:4]: Anyway, I think you can keep it as drafts.csswg.org.

**Comment by lens0021** [comment]
> Do we really need to have this when there is the corresponding MDN item? https://caniuse.com/mdn-css_types_color_alpha_hexadecimal_notation

Apologies, I was confused when I was looking for features, and I'm still confused. CONTRIBUTING.md does not seem to describe mdn and I don't know the difference between drafts.csswg.org and W3C working draft. Do you think I should close this PR?

**Comment by saschanaz** [comment]
I mean, should we consider this caniuse-specific item as duplicated? I see no point to have two items for the same feature.

**Comment by Fyrd** [comment]
I agree with keeping it as the drafts spec but updating it to `wd`. Thanks!

As for the duplicate status, I have marked the MDN entry as a duplicate of this one which means next time the MDN data is rebuilt it will no longer appear (since the caniuse entry can include more info)

**Comment by Fyrd** [comment]
Updated manually so closing this PR.

**Pull request closed without merging**